tests/db/migration_tests/test_v202505150_to_v202505260.py

```diff
@@ -0,0 +1,40 @@
+"""
+Copyright 2025 Inmanta
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Contact: code@inmanta.com
+"""
+
+import os
+import re
+from collections import abc
+
+import asyncpg
+import pytest
+
+file_name_regex = re.compile("test_v([0-9]{9})_to_v[0-9]{9}")
+part = file_name_regex.match(__name__)[1]
+
+
+@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), f"dumps/v{part}.sql"))
+async def test_add_claim_table(
+    postgresql_client: asyncpg.Connection,
+    migrate_db_from: abc.Callable[[], abc.Awaitable[None]],
+    get_tables_in_db: abc.Callable[[], abc.Awaitable[list[str]]],
+) -> None:
+    assert "role" not in await get_tables_in_db()
+    assert "role_assignment" not in await get_tables_in_db()
+    await migrate_db_from()
+    assert "role" in await get_tables_in_db()
+    assert "role_assignment" in await get_tables_in_db()
```

Comment:
I don't think we need any test for this. 

i.e. this doesn't really test anything that is not covered by other tests.